index.md

```diff
@@ -17,55 +20,79 @@ run `pipeline.wdl` using
 java -jar cromwell-<version>.jar run -i inputs.json pipeline.wdl
 ```
 
-The inputs JSON can be generated using WOMtools as described in the [WOMtools
-documentation](http://cromwell.readthedocs.io/en/stable/WOMtool/). Note that
-not some inputs should not be used! See [this page](inputs.md) for more
-information.
+### Inputs
+Inputs are provided through a JSON file. The primary inputs are described
```

Comment:
The primary inputs are described -> The minimally required inputs to get you started are described